nimare/utils.py

```diff
@@ -88,7 +89,7 @@ def dict_to_coordinates(data, masker, space):
             )
 
             # Optional information
-            for k in list(set(exp["coords"].keys()) - set(columns)):
+            for k in list(set(exp["coords"].keys()) - set(core_columns)):
```

Comment:
Is this a bug in the current version?

Reply:
yes, I believe so since when a non-essential column was added to `columns` (such as `z_stat`), the next time this comparison was performed, `columns` already contained `z_stat` and then the actual z_stat value for that study contrast was never added to the dataframe.

Reply:
Thanks for catching it then. Could you just add a note about it to the PR summary?